src/jQueryShim.js

```diff
@@ -72,21 +77,25 @@ const ajax = function(options) {
   return {
     abort: function(reason) {
       return request.abort(reason);
-    }
-  };
+    },
+    xhrSupported: (function() {
+      var xhrSupported = xhr();
+      return !!xhrSupported && ("withCredentials" in xhrSupported);
```

Comment:
You're right, also the code would not work. I missed the fact that `ajax` is a function, so this shouldn't be inside here in the first place, or it will always be `undefined` anyway.
I refactored the code.